sdk/core/amqp-common/src/errors.ts

```diff
@@ -526,6 +527,19 @@ export function isSystemError(err: any): boolean {
   return result;
 }
 
+export function isWebsocketError(err: any): boolean {
+  let result: boolean = false;
+  if (
+    !isNode &&
+    window !== undefined &&
```

Comment:
We should change this to `window != undefined` as that will check for both `null` and `undefined` whereas `!==`  checks only for `undefined`

Reply:
It seems like other instances of same check use `typeof window !== undefined`
I've updated this one to be same as well for time being.

Reply:
@bterlson @ramya-rao-a 
Should we update all references to use `!=` check? Or remove it / move it to within isNode check?
How about moving it out to a separate module as it's a useful check?
I recall discussing this in typescript meeting with I think @jeremymeng @jonathandturner @AlexGhiondea as well.

Reply:
We have a common definition of isNode in a few places already (ms-rest, amqp-common). Ideally !isNode should be sufficient. What is the purpose of the extra check on window?

Reply:
Am ok with a separate dep, strawman: `@azure/core-platform-check` that is shared and re-exported by `@azure/core-amqp` and `@azure/core-http`.

Reply:
Oh, I suppose we could create a separate issue for that. For now I'm including change in this PR to export the isNode check and update all checks in the package where necessary.

Reply:
We can also simplify this by replacing `!isNode && window !== undefined` with `!isNode && window`, because the only reason the check on `window` exists is to ensure `window.WebSocket` doesn't throw